torchaudio-contrib/io.py

```diff
@@ -0,0 +1,48 @@
+import torch
+import numpy as np
+import warnings
+from scipy.io import wavfile
+import librosa
+
+def load(filename, sr=44100, mono=True, norm=False, device='cpu', dtype=np.float32):
+    """
+    Generic wrapper for reading an audio file.
+    Different libraries offer different speeds for this, so this routine is the
+    'catch-all' for whatever read routine happens to work best
+
+    Tries a fast method via scipy first, reverts to slower librosa when necessary.
+    """
+    # first try to read via scipy, because it's fast
+    scipy_ok = False
+    with warnings.catch_warnings(record=True) as w:
+        warnings.simplefilter("error")    # scipy throws warnings which should be errors
+        try:
+            out_sr, signal = wavfile.read(filename)
+            scipy_ok = True
+        except wavfile.WavFileWarning:
+            print("read_audio_file: Warning raised by scipy. ",end="")
```

Comment:
As long as torch supports Python 2, we should too. Would need a `from __future__ import print_function`, or a way that does not need the `end` keyword argument. I mean, you could just print everything when you reach librosa.

Reply:
Good point. I haven't thought about Python 2 in a couple years.